pkg/server/home_workspaces.go

```diff
@@ -150,7 +157,7 @@ type localInformersAccess struct {
 	getClusterWorkspace   func(logicalcluster.Name) (*tenancyv1alpha1.ClusterWorkspace, error)
 	getClusterRole        func(lcluster logicalcluster.Name, name string) (*rbacv1.ClusterRole, error)
 	getClusterRoleBinding func(lcluster logicalcluster.Name, name string) (*rbacv1.ClusterRoleBinding, error)
-	getAPIBinding         func(clusterName logicalcluster.Name, name string) (*apisv1alpha1.APIBinding, error)
+	getTenancyAPIBinding  func(clusterName logicalcluster.Name) (*apisv1alpha1.APIBinding, error)
```

Comment:
what is it doing? not clear from it's name.

Reply:
getting the apibinding that is for the tenancy.kcp.dev apiexport